manifest.md

```diff
@@ -84,14 +84,22 @@ A client will distinguish a manifest list from an image manifest based on the Co
 - **`annotations`** *string-string hashmap*
 
     This OPTIONAL property contains arbitrary metadata for the manifest list.
-    Annotations is a key-value, unordered hashmap.
-    Keys are unique, and best practice is to namespace the keys.
+    Annotations MUST BE a key-value, unordered hashmap where both the key and value MUST be strings.
+    While the value MUST be present, it MAY be an empty string.
+    Keys MUST be unique, and best practice is to namespace the keys.
+    Keys using the `org.opencontainers` namespace are reserved and MUST NOT be used.
```

Comment:
I think we want to claim this [with a trailing period][1] and also SHOULD a [general rule for mapping a DNS name to the key namespace][2].

[1]: https://github.com/opencontainers/image-spec/pull/164#issuecomment-237625793
[2]: https://github.com/opencontainers/image-spec/pull/164#issuecomment-239182187